helm/tenant/templates/_helpers.tpl

```diff
@@ -20,6 +20,17 @@ If release name contains chart name it will be used as a full name.
   {{- end -}}
 {{- end -}}
 
+{{/*
+Allow the release namespace to be overridden for multi-namespace deployments in combined charts
+*/}}
+{{- define "minio-operator.namespace" -}}
```

Comment:
Now I remember, this is why I got confused earlier.
I feel we should call this variable `minio-tenant.namespace` as we are overriding the tenant namespace and not operator namespace.

Reply:
Good catch @shtripat I've fixed this.

Note: the other templates in this chart are names `minio-operator.`. This should be fixed in another PR probably